package-lock.json

```diff
@@ -6914,6 +6914,16 @@
       "integrity": "sha512-1Yj8h9Q+QDF5FzhMs/c9+6UntbD5MkRfRwac8DoEm9ZfUBZ7tZ55YcGVAzEe4bXsdQHEk+s9S5wsOKVdZrw0tQ==",
       "dev": true
     },
+    "bindings": {
```

Comment:
Should we remove this from this PR?
If we need to update we can do in another PR